src/integrations/admin/workouts-integration.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use WPSEO_Admin_Asset_Manager;
+use WPSEO_Shortlinker;
+use Yoast\WP\SEO\Conditionals\Admin_Conditional;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Models\Indexable;
+use Yoast\WP\SEO\Premium\Actions\Link_Suggestions_Action;
+use Yoast\WP\SEO\Premium\Helpers\Prominent_Words_Helper;
+use Yoast\WP\SEO\Repositories\Indexable_Repository;
+
+/**
+ * WorkoutsIntegration class
+ */
+class Workouts_Integration implements Integration_Interface {
+
+	/**
+	 * The admin asset manager.
+	 *
+	 * @var WPSEO_Admin_Asset_Manager
+	 */
+	private $admin_asset_manager;
+
+	/**
+	 * The shortlinker.
+	 *
+	 * @var WPSEO_Shortlinker
+	 */
+	private $shortlinker;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public static function get_conditionals() {
+		return [ Admin_Conditional::class ];
+	}
+
+	/**
+	 * Workouts_Integration constructor.
+	 *
+	 * @param WPSEO_Admin_Asset_Manager $admin_asset_manager The admin asset manager.
+	 * @param WPSEO_Shortlinker         $shortlinker         The shortlinker.
+	 */
+	public function __construct(
+		WPSEO_Admin_Asset_Manager $admin_asset_manager,
+		WPSEO_Shortlinker $shortlinker
+	) {
+		$this->admin_asset_manager = $admin_asset_manager;
+		$this->shortlinker         = $shortlinker;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function register_hooks() {
+		if ( ! YoastSEO()->helpers->product->is_premium() ) {
```

Comment:
Could be nice to turn this into a conditional or alternatively inject the `Product_Helper` directly.